examples/index.html

```diff
@@ -189,6 +189,26 @@ <h2 style="margin-top:20px">Save</h2>
 
   <footer>
     <hr style="margin:20px;"><center><a class="color:grey;" id="clear-cache">Clear offline cache</a></center>
+    <div class="row">
+      <div class="col-md-6">
+        <h2>Need Help?</h2>
+        <p>
+          Post a link to this and ask for help from other community members on <a href="https://publiclab.org/">PublicLab.org</a>
+        </p>
+        <p>
+          <a class="btn btn-primary" href="https://publiclab.org/questions">Ask a question &raquo;</a>
```

Comment:
I don't think linking to publiclab.org is good. Maybe link to GitHub issues? Or swap the ask a question section for something else?

Reply:
What is your idea about this? Is github issues good? Or should it link to something else?